--- a/drivers/gpu/drm/i915/display/intel_bios.c
+++ b/drivers/gpu/drm/i915/display/intel_bios.c
@@ -3793,8 +3793,16 @@
 {
 	struct intel_bios_encoder_data *devdata;

+	int ignoreEntry = 0;
+
 	list_for_each_entry(devdata, &display->vbt.display_devices, node)
-		func(display, devdata);
+	{
+		if (ignoreEntry != 1)
+		{
+			func(display, devdata);
+			ignoreEntry++;
+		}
+	}
 }

 static int intel_bios_vbt_show(struct seq_file *m, void *unused)